internal/bsr/chunk.go

```diff
@@ -18,6 +18,11 @@ const (
 	crcDataSize   = protocolSize + chunkTypeSize + directionSize + timestampSize
 	chunkBaseSize = lengthSize + crcDataSize
 	crcSize       = 4
+
+	// MaxChunkLength sets an upper bound on BSR chunk lengths
+	// Default to 64MB, as this is the limit for protobufs, which back our chunks
+	// TODO: Should this be configurable as an option?
+	MaxChunkLength = 64 * 100 * 1000
```

Comment:
I still think this should probably be named something like `MaxChunkDataLength`.